_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_cenc.cpp

```diff
@@ -129,18 +129,21 @@ namespace UMC_HEVC_DECODER
         for(; index < rps->getNumberOfNegativePictures() + rps->getNumberOfPositivePictures(); index++)
                 pocList[numRefPicSetStCurrBefore + numRefPicSetStCurrAfter++] = picParam->CurrPic.pic_order_cnt + rps->getDeltaPOC(index);
 
-        for(; index < rps->getNumberOfNegativePictures() + rps->getNumberOfPositivePictures() + rps->getNumberOfLongtermPictures(); index++)
+        if(!(supplier->GetDPBList()))
```

Comment:
Absence of DPB (null pointer) here is a logic error u can't just silently ignore it. I'd rather throw an exception.

Reply:
Hi,
I use "throw h265_exception(UMC_ERR_NULL_PTR)" here instead of just silently ignoreing it. Is that OK?